docs/data-sources/teams.md

```diff
@@ -0,0 +1,41 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "dagster_teams Data Source - dagster"
+subcategory: ""
+description: |-
+  Retrieve information about a Dagster Cloud teams.
```

Comment:
```suggestion
  Retrieve information about one or more Dagster Cloud teams.
```

Reply:
Maybe useful to mention that https://pkg.go.dev/regexp#MatchString is used to do the matching (that explains the syntax immediately)